.circleci/make.sh

```diff
@@ -6,32 +6,33 @@ PRE=config.pre.yml
 MID=config.mid.yml
 END=config.end.yml
 
-cat >>"${END}" <<END
-    - "examples-sink":
-        requires:
-END
+#cat >>"${END}" <<END
+#    - "examples-sink":
+#        requires:
+#END
+#
+#for CONN in ETH ALGO CFX ; do
+#  case "${CONN}" in
+#    ALGO) PER=8 ;;
+#    CFX) PER=8 ;;
+#    ETH) PER=16 ;;
+#  esac
+#  SIZE=$(((TOTAL + (PER - 1)) / PER))
+#  for RANK in $(seq 0 $((SIZE - 1))) ; do
+#    cat >>"${MID}" <<END
+#    - "examples":
+#        name: "examples.${CONN}.${RANK}"
+#        connector: "${CONN}"
+#        size: "${SIZE}"
+#        rank: "${RANK}"
+#        requires:
+#          - "build"
+#END
+#    cat >>"${END}" <<END
+#          - "examples.${CONN}.${RANK}"
+#END
+#  done
+#done
 
-for CONN in ETH ALGO CFX ; do
-  case "${CONN}" in
-    ALGO) PER=8 ;;
-    CFX) PER=8 ;;
-    ETH) PER=16 ;;
-  esac
-  SIZE=$(((TOTAL + (PER - 1)) / PER))
-  for RANK in $(seq 0 $((SIZE - 1))) ; do
-    cat >>"${MID}" <<END
-    - "examples":
-        name: "examples.${CONN}.${RANK}"
-        connector: "${CONN}"
-        size: "${SIZE}"
-        rank: "${RANK}"
-        requires:
-          - "build"
-END
-    cat >>"${END}" <<END
-          - "examples.${CONN}.${RANK}"
-END
-  done
-done
-
-cat "${PRE}" "${MID}" "${END}" > config.gen.yml
+cat "${PRE}" > config.gen.yml
```

Comment:
I think you should move this line up to 9 and add `exit 0` so it is more clear that this is just an experiment.